backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/select/internal/BpkSelectImpl.kt

```diff
@@ -56,7 +57,7 @@ internal fun BpkSelectImpl(
 ) {
     var expanded by remember { mutableStateOf(false) }
     val selectText = selectedIndex?.let { options.getOrNull(selectedIndex) } ?: ""
-
+    val focusManager = LocalFocusManager.current
```

Comment:
added this just cuz I find it annoying every time I interact with age selection and after selecting the ExposedDropdownMenuBox keeps highlighted, we can ignore this and not include it to the changes  if you think that's fine